### PR TITLE
Added trace_login_config config to trace JVM security config

### PR DESCRIPTION
**Pull request by gaborgsomogyi**
I've tested it the following way:
```
[gaborsomogyi:~/test] $ cat test.java 
class A {
  public void foo() {
    System.out.println("foo");
  }
}

class Test {
  public static void main(String[] args) {
    System.out.println("main");
    new A().foo();
  }
}
...
[gaborsomogyi:~/test] $ cat kafka_client_jaas.conf 
KafkaClient {
    com.sun.security.auth.module.Krb5LoginModule required
    useKeyTab=true
    storeKey=true
    keyTab="./kafka_client.keytab"
    useTicketCache=false
    serviceName="kafka"
    principal="user@MY.DOMAIN.COM";
};
...
[gaborsomogyi:~/trace-agent] trace_login_config* 3s ± cat src/main/resources/actions.txt 
trace_login_config A foo entry_name:KafkaClient
mvn clean package
```
And the result looks like this:
```
[gaborsomogyi:~/test] $ java -javaagent:/Users/gaborsomogyi/trace-agent/target/trace-agent-1.0-SNAPSHOT.jar -Djava.security.auth.login.config=./kafka_client_jaas.conf Test 
TraceAgent is initializing
TraceAgent tries to install actions: [{actionId='trace_login_config', classMatcher='A', methodMatcher='foo', actionArgs='entry_name:KafkaClient'}]
TraceAgent installed actions successfully
main
TraceAgent (trace_login_config): `public void A.foo() login config for entry "KafkaClient"
KafkaClient {
	com.sun.security.auth.module.Krb5LoginModule LoginModuleControlFlag: required
	principal=user@MY.DOMAIN.COM
	storeKey=true
	keyTab=./kafka_client.keytab
	useKeyTab=true
	useTicketCache=false
	serviceName=kafka
}

foo
```

### CODE REVIEW

**Comment by gaborgsomogyi** [comment]
cc @attilapiros

File: README.md [hunk 1]
```diff
@@ -304,6 +304,7 @@ Here is the full list of actions and supported `params`
 | trace_retval         | isDateLogged, log_threshold_ms                |
 | counter              | isDateLogged, count_frequency                 |
 | avg_timing           | isDateLogged, window_length                   |
+| trace_login_config   | isDateLogged, entry_name                      |
```
Comment (attilapiros) [inline README.md:307]: Please extend this documentation with the description of the new action (please refer to the javadoc of `javax.security.auth.login.Configuration` and show an example output).

**Comment by attilapiros** [comment]
Thanks for the contribution! LGTM

File: README.md (hunk 1 above)
Comment (gaborgsomogyi) [reply README.md:332]: Added.

**Pull request merged**